README.md

```diff
@@ -12,115 +16,134 @@ This server connects agents to your Elasticsearch data using the Model Context P
 
 ## Prerequisites
 
-* Node.js (v22+)
 * An Elasticsearch instance
 * Elasticsearch API key with appropriate permissions
-* Claude Desktop App (free version is sufficient)
+* MCP Client (e.g. Claude Desktop)
+
+## Demo
+
+https://github.com/user-attachments/assets/5dd292e1-a728-4ca7-8f01-1380d1bebe0c
 
 ## Installation & Setup
 
 ### Using the Published NPM Package
 
-The easiest way to use Elasticsearch MCP Server is through the published npm package:
+> [!TIP]
+> The easiest way to use Elasticsearch MCP Server is through the published npm package.
 
 1. **Configure Claude Desktop App**
-   - Open Claude Desktop App
-   - Go to Settings > Developer > MCP Servers
-   - Click `Edit Config` and add a new MCP Server with the following configuration to your `claude_desktop_config.json`:
+   - Open your MCP Client (Claude Desktop)
```

Comment:
suggestion: a link to other mcp clients https://modelcontextprotocol.io/clients

Reply:
++